docs/polygon_tutorial.ipynb

```diff
@@ -429,7 +433,13 @@
     "poly1 = h3.LatLngPoly([(37.804, -122.412), (37.778, -122.507), (37.733, -122.501)])\n",
     "poly2 = h3.LatLngPoly(\n",
     "    [(37.803, -122.408), (37.736, -122.491), (37.738, -122.380), (37.787, -122.39)],\n",
-    "    [(37.760, -122.441), (37.772, -122.427), (37.773, -122.404), (37.758, -122.401), (37.745, -122.428)]\n",
```

Comment:
I was about to complain about this, but its inconsistent with what I just said above. 😬